sources/packages/backend/apps/db-migrations/src/sql/User/Create-user-history-table.sql

```diff
@@ -0,0 +1,61 @@
+-- Table to keep historical data.
+-- Columns history_timestamp and history_operation should be default columns for all the history tables.
+-- Columns other than the above-mentioned should reflect the columns from the original table without constraints and relationships.
+-- The order of the columns below follow the exact same order from the existing table.
+CREATE TABLE sims.users_history (
+  history_timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+  history_operation VARCHAR(20) NOT NULL,
+  id INT,
+  user_name VARCHAR(300),
+  email VARCHAR(300),
+  first_name VARCHAR(100),
+  last_name VARCHAR(100),
+  created_at TIMESTAMP WITH TIME ZONE,
+  updated_at TIMESTAMP WITH TIME ZONE,
+  is_active BOOLEAN,
+  creator INT,
+  modifier INT,
+  identity_provider_type sims."identity_provider_types"
```

Comment:
Minor. For enums in other sqls we do not use the double quotes. Is there a particular reason here?

Reply:
No reason, it was generated from DBeaver and I forgot to change it. I will adjust it.

Reply:
Changed.